source/internal.d.ts

```diff
@@ -251,3 +251,10 @@ export type IsNotFalse<T extends boolean> = [T] extends [false] ? false : true;
 Returns a boolean for whether the given type is `null`.
 */
 export type IsNull<T> = [T] extends [null] ? true : false;
+
+type ObjectKey = string | number | symbol;
```

Comment:
Use the built-in `PropertyKey`

Reply:
Cheers, wasn't sure what it was called :)